### PR TITLE
Removed deprecated center tags

### LINKED ISSUE #44: Remove center tag
@hemangsk center is deprecated, the whole home.html looks very weird and not at all clean. E.g. you use `col-md-6` and `col-md-5` those two don't add up to 12 :P

### PR DESCRIPTION
**Pull request by damngamerz**
mentors.html:removed center tag
projects.html:removed center tag
Removed both the Tags as they were deprecated
Fixes #44

### CODE REVIEW

**Comment by sils** [comment]
@damngamerz did you test the changes? Can you post a screenshot of the affected pages?

**Comment by damngamerz** [comment]
Yes !! Here you go :+1: 
screenshot from 2017-01-30 14-51-20
screenshot from 2017-01-30 14-51-26

**Comment by damngamerz** [comment]
@sils should i do it ?
it just needed some changes in style.css
`.profile-img`

**Comment by sils** [comment]
@damngamerz well we can't merge it like this ;)

**Comment by damngamerz** [comment]
@sils Here you go !!! 
found 1 more center tag to be deprecated :)
mentors.html: Removed a center Tag
style.css: Modified .profile-img
screenshot from 2017-01-30 15-22-31

**Comment by NiklasMM** [comment]
ack  	e1a2fd0

File: partials/tabs/mentors.html [hunk 1]
```diff
@@ -47,7 +47,7 @@
             <div  ng-repeat="member in gic.adminsList" class="card card-main showcase-card-gi">
                 <div class="empty"></div>
                 <div class="card-action profile-data" >
-                  <center><img class="profile-img" height="100" ng-src="{{ member.github_avatar_url }}"></center>
+                  <img class="profile-img" height="100" ng-src="{{ member.github_avatar_url }}">
```
Comment (sils) [inline partials/tabs/mentors.html]: why not just apply the center-block class? Should be avail by default in bootstrap

File: partials/tabs/projects.html [hunk 2]
```diff
@@ -6,10 +6,8 @@
         <input ng-model="searchText" placeholder="Search for a project" id="search" type="text" class="validate">
       </div>
       <div class="input-field col-md-2">
-        <center>
-          <div><a class="explore waves-effect waves-light btn">Explore</a>
-          </div>
-        </center>
+        <div><a class="explore waves-effect waves-light btn">Explore</a>
+        </div>
```
Comment (sils) [inline partials/tabs/projects.html]: the opening and closing div is redundant too btw.

File: partials/tabs/mentors.html (hunk 1 above)
Comment (damngamerz) [inline partials/tabs/mentors.html]: yup it seems to do the work!! will fix up !!! and revert the changes done in style.css

File: partials/tabs/projects.html (hunk 2 above)
Comment (damngamerz) [inline partials/tabs/projects.html]: Removing this extra div tag :+1:

**Comment by damngamerz** [comment]
@sils Tested!! with center-block seems to do the work without adding anything in style.css
Last commit logs
projects.html: Removed extra div tag.
mentors.html: added center-block to specific places.
style.css: Reverted back the changes.

**Comment by sils** [comment]
ack https://github.com/coala/projects/pull/162/commits/918e9e8307b9f7daabcda4a8d3e59862bd285864 https://github.com/coala/projects/pull/162/commits/faef2524505593e736ee7973e261d2272e960ee1

**Comment by sils** [comment]
rebase needed!

**Comment by damngamerz** [comment]
@sils Found 1 more extra div tag Removed and Rebased.

File: partials/tabs/projects.html [hunk 3]
```diff
@@ -6,10 +6,7 @@
         <input ng-model="searchText" placeholder="Search for a project" id="search" type="text" class="validate">
       </div>
       <div class="input-field col-md-2">
```
Comment (hemangsk) [inline partials/tabs/projects.html]: Add a center-block here as well 👍 can you also post a screenshot of this page after changes

Comment (damngamerz) [inline partials/tabs/projects.html]: @hemangsk here u go!! It remains the same though.
should i commit these changes as well ?
screenshot from 2017-01-31 00-31-34

**Comment by arnab1896** [comment]
ack https://github.com/coala/projects/pull/162/commits/bf4f73249b40f6faec7260c6e8e1e91bd45fbcfd

**Comment by hemangsk** [comment]
@damngamerz Its good practice to have file names in the commit message 😃  
eg: 
```
filename: shortlog

commit body

Fixes/Closes
```
Your changes are perfect now and the commits are atomic as well 👍  👍  
Make sure to follow the commit message guidelines at [coala.io/commit](http://coala.io/commit)

**Comment by damngamerz** [comment]
@hemangsk Done :) :+1:

**Comment by hemangsk** [comment]
Regarding 963dfb4, why put home.html in commit message if it is not being modified at all, you can just have the commit message as Remove deprecated center tag

**Comment by damngamerz** [comment]
@hemangsk Damn!! I missed that was copy pasting thanks for pointing. Done! :+1:

**Comment by hemangsk** [comment]
ack 84eb429 70c9f20 74ccc23

**Comment by hemangsk** [comment]
@rultor merge
